proof/refine/AARCH64/ArchInvsDefs_H.thy

```diff
@@ -1,35 +1,32 @@
 (*
- * Copyright 2023, Proofcraft Pty Ltd
+ * Copyright 2024, Proofcraft Pty Ltd
```

Comment:
I thought we want to keep the earlier dates? If I'm wrong with that, you could also change the copyright in `InvariantsPre_H`.

Reply:
This is a new file, it just looks like it changed, because Raf set it up so there is a more useful diff. I guess it is debatable whether the content is really new if it's mostly moved, but there is also enough new stuff in here to actually call this new material from scratch.

Reply:
Hmm, I guess there's trade-offs. If we're thinking of it as new content should it keep the Data61 copyright?

Reply:
I believe so, I'm taking other people's stuff, rearranging it in a different way, and then adding my own stuff. So that bumps the Proofcraft copyright and keeps the one that made the old stuff.

Reply:
For my clarity, in general if a change is 'just' updating existing content then we don't want to bump the copyright, right? I know that there's a spectrum between updating and new content, but that was my understanding and what led to this comment initially.

Reply:
On the topic, how about `Invariants_H` in this PR? In my mind that one at least should keep the original copyright.

Reply:
It's hard to know what the threshold is. We're adding requalifies, rearranging stuff with the locales, renaming the locales ... @lsf37 ?

Reply:
My understanding is that how large a change to existing content is is relevant to whether we add the Proofcraft copyright or not, but that once the copyright is there we want to keep the original date. I might be thinking about this wrong of course, but I actually can't think of a situation where we want to bump a copyright. We could maybe make it a range and say something like `Copyright 2023-2024` but I don't know if that actually benefits us in anyway.

Reply:
Now that I've overthought this so much, I'm actually back to questioning the original copyright I commented on, in `ArchInvsDefs_H.thy`. We're either treating it as new content, in which case it shouldn't have the Data61 copyright, or we're considering it as based on the old content from `Invariants_H.thy` and it should keep the existing copyright.

Reply:
copyrights reverted